relayer/chains/cosmos/cosmos_chain_processor.go

```diff
@@ -361,7 +383,14 @@ func (ccp *CosmosChainProcessor) queryCycle(ctx context.Context, persistence *qu
 		ibcHeaderCache[heightUint64] = latestHeader
 		ppChanged = true
 
-		blockMsgs := ccp.ibcMessagesFromBlockEvents(blockRes.BeginBlockEvents, blockRes.EndBlockEvents, heightUint64)
+		base64Encoded := !ccp.chainProvider.GetTendermintEncodingBytes()
```

Comment:
I would expect a method named `GetTendermintEncodingBytes` to return a `[]byte` also having to use the not operator feels counterintuitive here. If this is the main use case for this method perhaps renaming to something more concise will eliminate the need to use `!` e.g. `TmEventsBase64Encoded` or something along those lines

Reply:
agreed, nice to remove the negations. I renamed it to `legacyTendermintEncoding`